### ISSUE TITLE
Build failing after clone

### ISSUE THREAD

**Issue by TheKnarf** [rickyhan/xprite-editor#5]
```
cargo run --bin xprite-native --release
error: failed to read `/Users/knarf/projects/xprite-editor/ase-rs/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
```

**Comment by TheKnarf** [comment]
Figured it out. I forgot to run `git submodule update --init --recursive`, perhaps this should be documented under the build instructions?

**Comment by rickyhan** [comment]
Closed by #6
